oximeter/collector/src/agent.rs

```diff
@@ -131,9 +141,49 @@ impl OximeterAgent {
             crate::results_sink::database_inserter(
```

Comment:
I might just add a comment above that this is specifically the task that inserts into the single-node database.

Reply:
Done in [95db1da](https://github.com/oxidecomputer/omicron/pull/7565/commits/95db1dac3725a65ee3c373ad40b40cf4e19e549f)